backend/src/main/java/de/otto/platform/gitactionboard/adapters/controller/BasicAuthenticationController.java

```diff
@@ -26,7 +26,7 @@
 @Slf4j
 @RestController
 @ConditionalOnExpression(
-    "T(org.springframework.util.StringUtils).hasText('${BASIC_AUTH_USER_DETAILS_FILE_PATH:}')")
+    "T(org.springframework.util.StringUtils).hasText('${BASIC_AUTH_USER_DETAILS_FILE_PATH:}') or T(org.springframework.util.StringUtils).hasText('${BASIC_AUTH_USER_DETAILS:}')")
```

Comment:
Please create a class to maintain this condition. Refer https://github.com/otto-de/gitactionboard/blob/main/backend/src/main/java/de/otto/platform/gitactionboard/config/security/DisableAuthentication.java